ecosystem

```diff
@@ -0,0 +1 @@
+devc
```

Comment:
not really because the name of the ecosystem is picked from the system file `/nail/etc/ecosystem` which is a standard practice across projects at yelp. Existence of that file as part of repo also serves as means of documentation.